### ISSUE TITLE
Arch linux fails due to keyring

### ISSUE THREAD

**Issue by dlasalle** [BytePackager/packagecore#125]
```
downloading required keys...
error: key "4A1AFC345EBE18F8" could not be looked up remotely
error: key "EEEEE2EEEE2EEEEE" could not be looked up remotely
error: required key missing from keyring
error: failed to commit transaction (unexpected error)
Errors occurred, no packages were upgraded.
Cleaning up self packagecore-5e7463203730272c.
packagecore-5e7463203730272c
packagecore-5e7463203730272c

###########################################################
# Failed to build package for 'archlinux'.
###########################################################
Traceback (most recent call last):
  File "/home/dominique/src/packagecore-master/packagecore/packager.py", line 160, in run
    imageName=build["dockerImage"])
  File "/home/dominique/src/packagecore-master/packagecore/packager.py", line 102, in _build
    recipe.build(container)
  File "/home/dominique/src/packagecore-master/packagecore/pkgbuild.py", line 211, in build
    container.execute(["/usr/bin/pacman", "--noconfirm", "-Su", "-yy"])
  File "/home/dominique/src/packagecore-master/packagecore/docker.py", line 149, in execute
    _checkedDockerCommand(["exec", self._name] + cmd)
  File "/home/dominique/src/packagecore-master/packagecore/docker.py", line 54, in _checkedDockerCommand
    raise DockerError("Command '%s' returned %d." % (cmd, status))
packagecore.docker.DockerError: Command '['exec', 'packagecore-5e7463203730272c', '/usr/bin/pacman', '--noconfirm', '-Su', '-yy']' returned 1.

###########################################################


```

**Comment by jarun** [comment]
I am getting affected by this too.

**Comment by dlasalle** [comment]
This particular issue is fixed in 0.9.7, but still issues with arch related to #128.
